app/src/main/java/org/fossasia/openevent/app/contract/model/IEventRepository.java

```diff
@@ -1,18 +1,22 @@
 package org.fossasia.openevent.app.contract.model;
 
+import org.fossasia.openevent.app.data.models.Attendee;
 import org.fossasia.openevent.app.data.models.Event;
 import org.fossasia.openevent.app.data.models.User;
 
 import java.util.List;
 
 import io.reactivex.Observable;
 
-public interface IEventModel {
+public interface IEventRepository {
```

Comment:
@iamareebjamal name interface as `EventDataSource` and let `EventDataRepository` implement it. `*DataSource` - `*DataRepository` (`*Source` for interface and `*Repository` for the class implementing it) naming convention is better. Also apply this to other models.